cmd/elkey/generate.go

```diff
@@ -0,0 +1,196 @@
+package main
+
+import (
+	"encoding/hex"
+	"fmt"
+	"math/rand"
+	"os"
+	"time"
+
+	"github.com/Layr-Labs/eigensdk-go/crypto/bls"
+
+	"github.com/Layr-Labs/eigensdk-go/crypto/ecdsa"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/google/uuid"
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	DefaultKeyFolder = "keys"
+
+	PasswordFile      = "password.txt"
+	PrivateKeyHexFile = "private_key_hex.txt"
+)
+
+var (
+	KeyTypeFlag = &cli.StringFlag{
+		Name:     "key-type",
+		Usage:    "key type to create (ecdsa or bls)",
+		Required: true,
+	}
+	NumKeysFlag = &cli.IntFlag{
+		Name:     "num-keys",
+		Usage:    "number of keys to create",
+		Required: true,
+	}
+	OutputDirFlag = &cli.StringFlag{
```

Comment:
I would personally prefer if we added a `--private-keys-file` and `--passwords-file` flags instead of needing to cd into the correct directory. Makes it more flexible and easier to use in bash scripts.

Reply:
this is just the output directory where keys will get created - check the `test_data/generated` folder for the directory structure

Reply:
I know. I’m saying you should add 2 extra flags to also make the input file location configurable

Reply:
there's no input file in this utility.  it creates password and private key file

Reply:
So it's easier for eigenda to read. they wanted both due to existing state of tests. logically you are right. we are just generating it so that it can be used both ways easily.

Reply:
discussed offline. this is due to DA needs